src/components/Pages/footer/Copyright.jsx

```diff
@@ -41,11 +42,36 @@ const Copyright = () => {
             <a>
               <BsFillRssFill />
             </a>
-          </div>
-        </div>
-      </div>
+          </Div>
+        </Div>
+      </Div>
     </>
   );
 };
+const Div = styled.div`
+  .footer .social-links a {
+    padding: 9px;
+    height: 40px;
+    width: 40px;
+    border-radius: 50%;
+    line-height: 40px;
+    text-align: center;
+    border: 2px solid #6c757d;
+    color: #6c757d;
+  }
+  .social-links a:hover {
+    background: #0d6efd;
+  }
 
+  .social-links {
+    text-align: end;
+  }
+  .footer {
+    border-top: 2px solid #44444426;
```

Comment:
The suggestion is to use `rgba` instead. Research first. Both the values are correct.